lib/sprig/seed/attribute.rb

```diff
@@ -54,7 +54,7 @@ def computed_value?(value)
       end
 
       def computed_value_regex
-        /<%[=]?(.*)%>/
+        /(<%[=]?(.*?)%>)/
```

Comment:
Why the `?` in the middle?  Shouldn't it just be `.*` or `.?` depending on if you want at least one character or not?